cmd/cli/info.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+
+	"github.com/spf13/cobra"
+)
+
+func isURL(arg string) bool {
+	_, err := url.ParseRequestURI(arg)
+	return err == nil
+}
+
+var infoCmd = &cobra.Command{
+	Use:   "info",
+	Short: "Provides informations about a puglin",
+	Long:  "Retrieves the .reef.yml from the repository and displays the informations.",
```

Comment:
You can use `Args:  cli.ExactArgs(2),` here to avoid having the test on `len()`

Reply:
Yeah, but this way I can validate the URL